src/worker/views/frivolitiesTeamSeasons.ts

```diff
@@ -315,6 +316,46 @@ const updateFrivolitiesTeamSeasons = async (
 				["most.value", "mov"],
 				["desc", "asc"],
 			];
+		} else if (type === "old_champ") {
+			title = "Oldest Championship Teams";
+			description = "These are oldest teams that won the title.";
+			extraCols.push(
+				{
+					key: "avgAge",
+					colName: "AvgAge",
+				},
+				{
+					key: "seed",
+					colName: "Seed",
+				},
+				{
+					key: ["most", "roundsWonText"],
+					keySort: "playoffRoundsWon",
+					colName: "Rounds Won",
+				},
+			);
+
+			filter = ts =>
+				!!ts.avgAge &&
+				ts.playoffRoundsWon >= 0 &&
+				(season > ts.season || phase > PHASE.PLAYOFFS);
+			getValue = ts => {
+				const roundsWonText = getRoundsWonText(ts);
+
+				// Keep in sync with helpers.roundsWonText
+				const validTexts = ["League champs"];
+				if (!validTexts.includes(roundsWonText)) {
+					return;
+				}
+				return {
+					value: ts.avgAge ? ts.avgAge : 0,
```

Comment:
Same thing here, but even more meaningless since this sets it to 0 anyway. But just general rule, don't put something that could inadvertently be falsy in a place like this. either do `ts.avgAge !== undefined` or, even better, use fancy modern syntax like `ts.avgAge ?? 0` which does not have the same falsy problem.